core/src/main/java/org/apache/carbondata/core/mutate/CarbonUpdateUtil.java

```diff
@@ -427,6 +427,10 @@ public static void cleanUpDeltaFiles(CarbonTable table, boolean forceDelete) {
 
     String validUpdateStatusFile = "";
 
+    boolean getOnlyAbortedFiles = true;
```

Comment:
It is hard to understand a Boolean variable with name start with `get`, can you rename it `isXXX` and add comment for it. The same for below variable